public/service_portal/v0.1.0/openapi.json

```diff
@@ -428,6 +428,49 @@
                 }
             }
         },
+        "/portfolio_items/{portfolio_item_id}/icon": {
+            "get": {
+                "tags": [
+                    "users",
+                    "admins"
+                ],
+                "summary": "Fetches the specified portfolio item's icon information, reaching out to topology if necessary",
+                "operationId": "fetchServiceOfferingIconFromPortfolioItem",
+                "description": "Fetch the specified portfolio item's icon information.\n",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/PortfolioItemID"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Portfolio Item Icon",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/components/schemas/ServiceOfferingIcon"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "401": {
+                        "$ref": "#/components/responses/Unauthorized"
+                    },
+                    "403": {
+                        "$ref": "#/components/responses/Forbidden"
+                    },
+                    "404": {
+                        "description": "Portfolio Item Not found"
+                    },
+                    "500": {
+                        "description": "Could not access the Topology Service"
```

Comment:
I'm assuming there is precedent for this error? I'm just not sure we need to name the Topology Service, but I could be convinced otherwise.